lib/whitehall_adapter.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class WhitehallAdapter
```

Comment:
It'd be good if this was more consistent with a GDS API Adapter. Perhaps subclassing `GdsApi::Base` and putting it in `lib/gds_api` would help bring consistency. 

It's also worth considering is this better in GDS API Adapters overall?

Reply:
I've created a `GdsApi::Whitehall` adapter, with the relevant methods within `content-publisher`.  I haven't moved it to gds-api-adapters since it feels that the only use for accessing Whitehall is from this application.